diff -up system-config-display-2.2/src/xserver.py.jx system-config-display-2.2/src/xserver.py
--- system-config-display-2.2/src/xserver.py.jx	2009-09-21 13:21:10.000000000 -0400
+++ system-config-display-2.2/src/xserver.py	2010-07-27 15:24:58.000000000 -0400
@@ -135,11 +135,6 @@ class XServer:
             return (True, END)
 
     def _run(self, xStartedCB = None):
-        def chroot():
-            if self.root != "/":
-                os.chroot(self.root)
-                os.chdir("/")
-
         args = ['-logfile', self.logfile, self.display] + self.serverflags
 
         if self.hwstate.videocard == None:
@@ -160,7 +155,7 @@ class XServer:
 
 	    old_sigusr1 = signal.signal(signal.SIGUSR1, sigusr1_handler)
 	    old_sigchld = signal.signal(signal.SIGCHLD, sigchld_handler)
-            proc = subprocess.Popen(["/usr/bin/Xorg"] + args, preexec_fn=chroot,
+            proc = subprocess.Popen(["/usr/bin/Xorg"] + args,
                                     stdout=noOutput, stderr=noOutput,
 				    preexec_fn=preexec_fn)
 	    signal.pause()
